README.md

```diff
@@ -1,24 +1,79 @@
+# in https://sonarcloud.io/documentation/project-administration/narrowing-the-focus/
+sonar.sources=.
+
+# Encoding of the source code. Default is default system encoding
+#sonar.sourceEncoding=UTF-8
+```
+The workflow, usually declared in `.codecatalyst/workflows/build.yaml`, looks like:
+
+```yaml
+Name: SonarCloudAnalysis
+SchemaVersion: "1.0"
+
+# Optional - Set automatic triggers.
+Triggers:
+  - Type: Push
+# Required - Define action configurations.
+Actions:
+  SonarCloudScanAction:
+    # Identifies the action. Do not modify this value.
+    Identifier: sonar/sonarcloud-scan@v1.0.0
+    # Specifies the source and/or artifacts to pass to the action as input.
+    Inputs:
+      # Required
+      Sources:
+        - WorkflowSource # This specifies that the action requires this Workflow as a source
+    Compute:
+      Type: EC2
+    # Defines the action's properties.
+    Configuration:
+      SonarToken: ${Secrets.SONAR_TOKEN}
+```
+
+When your workflow runs, the **SonarCloud Scan** will execute and send analysis results to [SonarCloud](https://sonarcloud.io/).
+
+
+## Configuration parameters
+
+- `SonarToken` – **Required** this is the token used to authenticate access to SonarCloud. You can generate a token on your [Security page in SonarCloud](https://sonarcloud.io/account/security/). You can set the `SONAR_TOKEN` environment variable in the "Secrets" settings page of your repository.
+- `SonarOrganization` – **Optional** The key of the SonarCloud organization to which the project belongs. If not provided, the organization key will be read from the `sonar-project.properties` file.
+- `SonarProjectKey` – **Optional** The key of the project on SonarCloud. If not provided, the project key will be read from the `sonar-project.properties` file.
+- `SonarBranchName` – **Optional** The name of the branch being analyzed.
+- `SonarProjectBaseDir` – **Optional** The base directory of the project to analyze. If not provided, the base directory will be the root of the repository.`
+- `SonarAdditionalArgs` – **Optional** Additional arguments to pass to the SonarScanner CLI. For example, `-Dsonar.verbose=true` to enable verbose logging.
 
 ## Do not use this CodeCatalyst action if you are in the following situations
 
-* Your code is built with Maven: run 'sonar:sonar' during the build
-* Your code is built with Gradle: use the Sonar plugin for Gradle during the build
+* Your code is built with Maven: run 'org.sonarsource.scanner.maven:sonar' during the build
+* Your code is built with Gradle: use the SonarQube plugin for Gradle during the build
 * You want to analyze a .NET solution: Follow our interactive tutorial for other CI's
 * You want to analyze C and C++ code: rely on our [SonarCloud Scan for C and C++](https://github.com/marketplace/actions/sonarcloud-scan-for-c-and-c) and look at [our sample C and C++ project](https://github.com/sonarsource-cfamily-examples?q=gh-actions-sc&type=all&language=&sort=)
```

Comment:
I think this is still true actually. People can keep using our github action for C/C++ thanks to the `aws/github-actions-runner` action